Tutorials/Creating-Basic-Site/Master-Template-The-Navigation-Menu/index-v8.md

```diff
@@ -0,0 +1,24 @@
+---
+versionFrom: 8.0.0
+---
+# Master Template - The Navigation Menu
+
+Now let's fix the navigation menu - there are two ways of doing this, you could have Umbraco dynamically create a navigation menu from the pages it has in the Content Tree, so that when an editor creates a page it automatically appears or, more simply you can hardcode it. 
```

Comment:
```suggestion
Now let's fix the navigation menu - there are two ways of doing this: 

1. You could have Umbraco dynamically create a navigation menu from the pages it has in the Content Tree, so that when an editor creates a page it automatically appears or, 
2. You can hardcode it. 
```